lib/zkevm_node.star

```diff
@@ -21,24 +21,22 @@ def _start_node_component(
     ]
     if http_api:
         cmd.append("--http.api=" + http_api)
-    return plan.add_service(
-        name=name,
-        config=ServiceConfig(
-            image=image,
-            ports=ports,
-            files={
-                "/etc/zkevm": config_files,
-            },
-            entrypoint=["/app/zkevm-node"],
-            cmd=cmd,
-        ),
+    return ServiceConfig(
+        image=image,
+        ports=ports,
+        files={
+            "/etc/zkevm": config_files,
+        },
+        entrypoint=["/app/zkevm-node"],
+        cmd=cmd,
     )
 
 
+# The synchronizer is required to run before any other zkevm node component.
+# This is why this component does not have a `create_service_config` method.
```

Comment:
this is great context for code maintainers ! nit: perhaps add more contextual comments like thi to improve readability and logic comprehension for future devs